packages/codegen.go/src/helpers.ts

```diff
@@ -309,10 +309,15 @@ export function formatValue(paramName: string, type: go.PossibleType, imports: I
 // returns the clientDefaultValue of the specified param.
 // this is usually the value in quotes (i.e. a string) however
 // it could also be a constant.
-export function formatLiteralValue(value: go.LiteralValue): string {
+// specify withCast: false for doc comments
```

Comment:
If `withCast` is relevant only to doc comments, maybe it should be e.g. `docComment` instead

Reply:
It's true at present, but I suppose it might not be in the future. Maybe I should just remove the comment.